[2023-10-03T11:33:10.042+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_data.get_data scheduled__2023-10-02T00:00:00+00:00 [queued]>
[2023-10-03T11:33:10.058+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_data.get_data scheduled__2023-10-02T00:00:00+00:00 [queued]>
[2023-10-03T11:33:10.059+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-03T11:33:10.083+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): get_data> on 2023-10-02 00:00:00+00:00
[2023-10-03T11:33:10.091+0000] {standard_task_runner.py:57} INFO - Started process 71 to run task
[2023-10-03T11:33:10.105+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'process_data', 'get_data', 'scheduled__2023-10-02T00:00:00+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/data_retrieval.py', '--cfg-path', '/tmp/tmpjt9sc0na']
[2023-10-03T11:33:10.113+0000] {standard_task_runner.py:85} INFO - Job 114: Subtask get_data
[2023-10-03T11:33:10.230+0000] {task_command.py:415} INFO - Running <TaskInstance: process_data.get_data scheduled__2023-10-02T00:00:00+00:00 [running]> on host eb8068a047da
[2023-10-03T11:33:10.367+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_data' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-02T00:00:00+00:00'
[2023-10-03T11:33:10.384+0000] {base.py:73} INFO - Using connection ID 'pgdatabase' for task execution.
[2023-10-03T11:33:10.433+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-03T11:33:10.449+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=process_data, task_id=get_data, execution_date=20231002T000000, start_date=20231003T113310, end_date=20231003T113310
[2023-10-03T11:33:10.479+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-03T11:33:10.504+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
